### PR TITLE
Add Brainfuck Command

### LINKED ISSUE #170: Brainfuck command
`s!brainfuck ++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++.`
Runs brainfuck code

### PR DESCRIPTION
**Pull request by upedd**
closes #170
btw its my third pull request in October 😉

### CODE REVIEW

**Comment by upedd** [comment]
image

File: src/commands/utility/brainfuck.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          required: false
+        }
+      ]
+    }, client)
+  }
+  constructOutputMsg (input, t) {
+    const output = `\`\`\`${input.output}\`\`\``
+    return `**${t('commands:brainfuck.steps')}:** ${input.steps}\n**${t('commands:brainfuck.time')}:** ${input.time}ms\n**${t('commands:brainfuck.output')}:** ${output}`
+  }
+  run ({ t, channel }, code, input) {
```
Comment (pedrofracassi) [inline src/commands/utility/brainfuck.js:29]: Add a line between functions please
```suggestion
  }
  
  constructOutputMsg (input, t) {
    const output = `\`\`\`${input.output}\`\`\``
    return `**${t('commands:brainfuck.steps')}:** ${input.steps}\n**${t('commands:brainfuck.time')}:** ${input.time}ms\n**${t('commands:brainfuck.output')}:** ${output}`
  }
  
  run ({ t, channel }, code, input) {
```

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1775,5 +1775,14 @@
     "nis": "NIS",
     "amount": "Amount",
     "city": "City"
+  },
+  "brainfuck": {
+    "commandDescription": "Executes a brainfuck code",
+    "commandUsage": "<code> <input optional>",
+    "noCode": "You have to give me a brainfuck code",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1782]:
```suggestion
    "noCode": "You have to give me some brainfuck code",
```

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -1775,5 +1775,14 @@
     "nis": "NIS",
     "amount": "Amount",
     "city": "City"
+  },
+  "brainfuck": {
+    "commandDescription": "Executes a brainfuck code",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1780]:
```suggestion
    "commandDescription": "Executes brainfuck code",
```

**Review by pedrofracassi — DISMISSED** [review]

**Comment by upedd** [comment]
These lines breaks broke CI 😢

**Comment by almeidx** [comment]
> These lines breaks broke CI 😢

**Comment by upedd** [comment]
> > These lines breaks broke CI 😢
> 
> 

Yeah these are the lines requested by @pedrofracassi

**Comment by almeidx** [comment]
> > > These lines breaks broke CI 😢
> > 
> > 
> > 
> 
> Yeah these are the lines requested by @pedrofracassi

I'm aware. Just remove the spaces in the empty lines

**Comment by upedd** [comment]
Fixed

**Review by almeidx — DISMISSED** [review]
LGTM

**Comment by almeidx** [comment]
Resolved conflicts

**Review by almeidx — APPROVED** [review]

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**